forem/forem
# Tag Page partial UI updates

## What type of PR is this? (check all applicable)


## Description
In this PR we've updated the tags page UI partially. We've done the following: 
- Removed the color strip at the top in favour of the # being colored. 
- Have the buttons always be at the bottom of the card
- Remove the rotation of the badge and allow it to be inline with the button. As a result also reduce the size of the badge. 
- Change the Title from Top Tags to Tags 
- Add a formatted published count
- Move the post count to the top right and remove the word published. 


**_Things that have not been implemented in this PR:_**
**_- I've not made all the changes to the buttons (i.e. their style and color), especially the secondary button as this is a bigger change that comes with some gotchas that I will outline in the issue._** 

## Related Tickets & Documents

- Related Issue #https://github.com/forem/forem/issues/19675
- Closes

## QA Instructions, Screenshots, Recordings

Load up the /tags page and ensure that the UI changes look fine. 

Before:
<img width="1247" alt="Screenshot 2023-07-31 at 11 23 04" src="https://github.com/forem/forem/assets/2786819/5a19c8bf-6dd4-4b72-a7da-b49fde05fbee">

After: (On prod these tags will have descriptions!)
 
<img width="1512" alt="Screenshot 2023-08-04 at 13 50 30" src="https://github.com/forem/forem/assets/2786819/9a8df101-1b54-495d-baa5-2ff5af6315fc">


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [x] No, and this is why: UI changes

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/7gaLHn9ap6aNq/giphy.gif)
